nemo/src/rule_model/components/import_export/file_formats.rs

```diff
@@ -149,6 +149,20 @@ pub enum FileFormat {
     ]))]
     #[assoc(arity = 4)]
     TriG,
+    /// SPARQL 1.1 
+    /// Is sparql a file format?
```

Comment:
The answer is no, but treating it as one fits quite well. We should likely refactor this (in a later PR) to `resource_formats`, e.g.?